### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by Miguel-Hombrados** [cornellius-gp/gpytorch#1877]
Is there a constant/bias kernel defined?
The equivalent to ".gaussian_process.kernels.ConstantKernel" in sklearn

Thank you!

**Comment by wjmaddox** [comment]
Not really, although it looks like from the [documentation of sklearn](https://scikit-learn.org/stable/modules/generated/sklearn.gaussian_process.kernels.ConstantKernel.html) that you can use it as either an outputscale for the kernel analogous to gpytorch's `ScaleKernel` or as an offset to the mean, analogous to gpytorch's `ConstantMean`.
